dmutils/email.py

```diff
@@ -91,16 +87,16 @@ def send_email(to_email_addresses, email_body, subject, from_email, from_name, r
         return_address = current_app.config.get('DM_EMAIL_RETURN_ADDRESS')
 
         result = email_client.send_email(
-            Source=u"{} <{}>".format(from_name, from_email),
+            Source="{} <{}>".format(from_name, from_email),
             Destination=destination_addresses,
             Message={
                 'Subject': {
-                    'Data': subject,
+                    'Data': subject.decode(),
```

Comment:
No need to decode()

Reply:
@tobiasbrown should we remove `[subject = to_bytes(subject](https://github.com/AusDTO/dto-digitalmarketplace-utils/pull/111/files/07fdee7c261c0770faf3ce5bfc4b7f9905247e88#diff-5c6407d010068da43fa9036bfd938dae7c9befeb2ea62fcee7c87fb21e7b3efeR71)`) that way we don't need to add the decode function.

Reply:
Let's leave this as is.  Emails have been received during testing.